### ISSUE TITLE
How to run code?

### ISSUE THREAD

**Issue by greatpak** [raj-nandu/Breat-Cancer-Complete-AI-Model-And-Web-Application#1]
Please guide me How i run your code?
Where is database file. Please help me

**Comment by ayungavis** [comment]
> 
> 
> Please guide me How i run your code?
> Where is database file. Please help me

Yes, the database file didn't found. Can you give me the database, please?
